.github/workflows/publish.yml

```diff
@@ -0,0 +1,40 @@
+name: Publish
+
+on:
+  push:
+    branches: [ main ]
```

Comment:
If I'm right, the `if` below ensures that publish is done only if triggered by a tag push, maybe we can only do 
```
on:
  push:
    tags: [ 'v*' ]
```

Reply:
Yes. But this is actually a general pattern. From a CI/CD perspective, the publishing workflow can be triggered by either a tag, a push to main or a pull_request. If you add a new job here, you won't have to worry about setting more triggers.

@amimart your opinion.

Reply:
Ah yes so we can leave the triggers for the sake of consistency between the CIs.
Thanks

Reply:
At least, we could remove the pull request trigger as we don't use it anymore on publish workflows.